<!doctype html>
<html lang=en>
  <head>
    <title>FileNotFoundError: [Errno 2] No such file or directory: '/Users/stuff/Desktop/Files/project/vton/fifa-demo/fifa-demo/results/test/try-on/tryon_with_bg.png'
 // Werkzeug Debugger</title>
    <link rel="stylesheet" href="?__debugger__=yes&amp;cmd=resource&amp;f=style.css">
    <link rel="shortcut icon"
        href="?__debugger__=yes&amp;cmd=resource&amp;f=console.png">
    <script src="?__debugger__=yes&amp;cmd=resource&amp;f=debugger.js"></script>
    <script>
      var CONSOLE_MODE = false,
          EVALEX = true,
          EVALEX_TRUSTED = false,
          SECRET = "NcYcES9Pgh5N2vYPwtqc";
    </script>
  </head>
  <body style="background-color: #fff">
    <div class="debugger">
<h1>FileNotFoundError</h1>
<div class="detail">
  <p class="errormsg">FileNotFoundError: [Errno 2] No such file or directory: &#39;/Users/stuff/Desktop/Files/project/vton/fifa-demo/fifa-demo/results/test/try-on/tryon_with_bg.png&#39;
</p>
</div>
<h2 class="traceback">Traceback <em>(most recent call last)</em></h2>
<div class="traceback">
  <h3></h3>
  <ul><li><div class="frame" id="frame-140357967907072">
  <h4>File <cite class="filename">"/Users/stuff/anaconda3/envs/image_based_vton/lib/python3.8/site-packages/flask/app.py"</cite>,
      line <em class="line">2551</em>,
      in <code class="function">__call__</code></h4>
  <div class="source library"><pre class="line before"><span class="ws">    </span>def __call__(self, environ: dict, start_response: t.Callable) -&gt; t.Any:</pre>
<pre class="line before"><span class="ws">        </span>&#34;&#34;&#34;The WSGI server calls the Flask application object as the</pre>
<pre class="line before"><span class="ws">        </span>WSGI application. This calls :meth:`wsgi_app`, which can be</pre>
<pre class="line before"><span class="ws">        </span>wrapped to apply middleware.</pre>
<pre class="line before"><span class="ws">        </span>&#34;&#34;&#34;</pre>
<pre class="line current"><span class="ws">        </span>return self.wsgi_app(environ, start_response)</pre></div>
</div>

<li><div class="frame" id="frame-140357967906960">
  <h4>File <cite class="filename">"/Users/stuff/anaconda3/envs/image_based_vton/lib/python3.8/site-packages/flask/app.py"</cite>,
      line <em class="line">2531</em>,
      in <code class="function">wsgi_app</code></h4>
  <div class="source library"><pre class="line before"><span class="ws">            </span>try:</pre>
<pre class="line before"><span class="ws">                </span>ctx.push()</pre>
<pre class="line before"><span class="ws">                </span>response = self.full_dispatch_request()</pre>
<pre class="line before"><span class="ws">            </span>except Exception as e:</pre>
<pre class="line before"><span class="ws">                </span>error = e</pre>
<pre class="line current"><span class="ws">                </span>response = self.handle_exception(e)</pre>
<pre class="line after"><span class="ws">            </span>except:  # noqa: B001</pre>
<pre class="line after"><span class="ws">                </span>error = sys.exc_info()[1]</pre>
<pre class="line after"><span class="ws">                </span>raise</pre>
<pre class="line after"><span class="ws">            </span>return response(environ, start_response)</pre>
<pre class="line after"><span class="ws">        </span>finally:</pre></div>
</div>

<li><div class="frame" id="frame-140357967906064">
  <h4>File <cite class="filename">"/Users/stuff/anaconda3/envs/image_based_vton/lib/python3.8/site-packages/flask/app.py"</cite>,
      line <em class="line">2528</em>,
      in <code class="function">wsgi_app</code></h4>
  <div class="source library"><pre class="line before"><span class="ws">        </span>ctx = self.request_context(environ)</pre>
<pre class="line before"><span class="ws">        </span>error: t.Optional[BaseException] = None</pre>
<pre class="line before"><span class="ws">        </span>try:</pre>
<pre class="line before"><span class="ws">            </span>try:</pre>
<pre class="line before"><span class="ws">                </span>ctx.push()</pre>
<pre class="line current"><span class="ws">                </span>response = self.full_dispatch_request()</pre>
<pre class="line after"><span class="ws">            </span>except Exception as e:</pre>
<pre class="line after"><span class="ws">                </span>error = e</pre>
<pre class="line after"><span class="ws">                </span>response = self.handle_exception(e)</pre>
<pre class="line after"><span class="ws">            </span>except:  # noqa: B001</pre>
<pre class="line after"><span class="ws">                </span>error = sys.exc_info()[1]</pre></div>
</div>

<li><div class="frame" id="frame-140357967906848">
  <h4>File <cite class="filename">"/Users/stuff/anaconda3/envs/image_based_vton/lib/python3.8/site-packages/flask/app.py"</cite>,
      line <em class="line">1825</em>,
      in <code class="function">full_dispatch_request</code></h4>
  <div class="source library"><pre class="line before"><span class="ws">            </span>request_started.send(self)</pre>
<pre class="line before"><span class="ws">            </span>rv = self.preprocess_request()</pre>
<pre class="line before"><span class="ws">            </span>if rv is None:</pre>
<pre class="line before"><span class="ws">                </span>rv = self.dispatch_request()</pre>
<pre class="line before"><span class="ws">        </span>except Exception as e:</pre>
<pre class="line current"><span class="ws">            </span>rv = self.handle_user_exception(e)</pre>
<pre class="line after"><span class="ws">        </span>return self.finalize_request(rv)</pre>
<pre class="line after"><span class="ws"></span> </pre>
<pre class="line after"><span class="ws">    </span>def finalize_request(</pre>
<pre class="line after"><span class="ws">        </span>self,</pre>
<pre class="line after"><span class="ws">        </span>rv: t.Union[ft.ResponseReturnValue, HTTPException],</pre></div>
</div>

<li><div class="frame" id="frame-140357967906736">
  <h4>File <cite class="filename">"/Users/stuff/anaconda3/envs/image_based_vton/lib/python3.8/site-packages/flask/app.py"</cite>,
      line <em class="line">1823</em>,
      in <code class="function">full_dispatch_request</code></h4>
  <div class="source library"><pre class="line before"><span class="ws"></span> </pre>
<pre class="line before"><span class="ws">        </span>try:</pre>
<pre class="line before"><span class="ws">            </span>request_started.send(self)</pre>
<pre class="line before"><span class="ws">            </span>rv = self.preprocess_request()</pre>
<pre class="line before"><span class="ws">            </span>if rv is None:</pre>
<pre class="line current"><span class="ws">                </span>rv = self.dispatch_request()</pre>
<pre class="line after"><span class="ws">        </span>except Exception as e:</pre>
<pre class="line after"><span class="ws">            </span>rv = self.handle_user_exception(e)</pre>
<pre class="line after"><span class="ws">        </span>return self.finalize_request(rv)</pre>
<pre class="line after"><span class="ws"></span> </pre>
<pre class="line after"><span class="ws">    </span>def finalize_request(</pre></div>
</div>

<li><div class="frame" id="frame-140357967429184">
  <h4>File <cite class="filename">"/Users/stuff/anaconda3/envs/image_based_vton/lib/python3.8/site-packages/flask/app.py"</cite>,
      line <em class="line">1799</em>,
      in <code class="function">dispatch_request</code></h4>
  <div class="source library"><pre class="line before"><span class="ws">            </span>and req.method == &#34;OPTIONS&#34;</pre>
<pre class="line before"><span class="ws">        </span>):</pre>
<pre class="line before"><span class="ws">            </span>return self.make_default_options_response()</pre>
<pre class="line before"><span class="ws">        </span># otherwise dispatch to the handler for that endpoint</pre>
<pre class="line before"><span class="ws">        </span>view_args: t.Dict[str, t.Any] = req.view_args  # type: ignore[assignment]</pre>
<pre class="line current"><span class="ws">        </span>return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)</pre>
<pre class="line after"><span class="ws"></span> </pre>
<pre class="line after"><span class="ws">    </span>def full_dispatch_request(self) -&gt; Response:</pre>
<pre class="line after"><span class="ws">        </span>&#34;&#34;&#34;Dispatches the request and on top of that performs request</pre>
<pre class="line after"><span class="ws">        </span>pre and postprocessing as well as HTTP exception catching and</pre>
<pre class="line after"><span class="ws">        </span>error handling.</pre></div>
</div>

<li><div class="frame" id="frame-140357976903568">
  <h4>File <cite class="filename">"/Users/stuff/Desktop/Files/project/vton/fifa-demo/app.py"</cite>,
      line <em class="line">367</em>,
      in <code class="function">virtual_tryon</code></h4>
  <div class="source "><pre class="line before"><span class="ws">    </span># print(&#39;Resized image in {}s&#39;.format(resize_time - start_time))</pre>
<pre class="line before"><span class="ws">    </span>#</pre>
<pre class="line before"><span class="ws">    </span># composite_background(&#39;Data_preprocessing/test_img/person.png&#39;,</pre>
<pre class="line before"><span class="ws">    </span>#                      &#39;results/test/try-on/person.png&#39;, img_mask)</pre>
<pre class="line before"><span class="ws"></span> </pre>
<pre class="line current"><span class="ws">    </span>response2 = send_file(</pre>
<pre class="line after"><span class="ws">        </span>&#39;fifa-demo/results/test/try-on/tryon_with_bg.png&#39;,</pre>
<pre class="line after"><span class="ws">        </span># attachment_filename=&#39;image2.png&#39;,</pre>
<pre class="line after"><span class="ws">        </span>mimetype=&#39;image/jpeg&#39;</pre>
<pre class="line after"><span class="ws">    </span>)</pre>
<pre class="line after"><span class="ws"></span> </pre></div>
</div>

<li><div class="frame" id="frame-140357976902672">
  <h4>File <cite class="filename">"/Users/stuff/anaconda3/envs/image_based_vton/lib/python3.8/site-packages/flask/helpers.py"</cite>,
      line <em class="line">537</em>,
      in <code class="function">send_file</code></h4>
  <div class="source library"><pre class="line before"><span class="ws">        </span>The ``add_etags``, ``cache_timeout`` and ``conditional``</pre>
<pre class="line before"><span class="ws">        </span>parameters were added. The default behavior is to add etags.</pre>
<pre class="line before"><span class="ws"></span> </pre>
<pre class="line before"><span class="ws">    </span>.. versionadded:: 0.2</pre>
<pre class="line before"><span class="ws">    </span>&#34;&#34;&#34;</pre>
<pre class="line current"><span class="ws">    </span>return werkzeug.utils.send_file(  # type: ignore[return-value]</pre>
<pre class="line after"><span class="ws">        </span>**_prepare_send_file_kwargs(</pre>
<pre class="line after"><span class="ws">            </span>path_or_file=path_or_file,</pre>
<pre class="line after"><span class="ws">            </span>environ=request.environ,</pre>
<pre class="line after"><span class="ws">            </span>mimetype=mimetype,</pre>
<pre class="line after"><span class="ws">            </span>as_attachment=as_attachment,</pre></div>
</div>

<li><div class="frame" id="frame-140357976902896">
  <h4>File <cite class="filename">"/Users/stuff/anaconda3/envs/image_based_vton/lib/python3.8/site-packages/werkzeug/utils.py"</cite>,
      line <em class="line">440</em>,
      in <code class="function">send_file</code></h4>
  <div class="source library"><pre class="line before"><span class="ws">        </span>if _root_path is not None:</pre>
<pre class="line before"><span class="ws">            </span>path = os.path.join(_root_path, path_or_file)</pre>
<pre class="line before"><span class="ws">        </span>else:</pre>
<pre class="line before"><span class="ws">            </span>path = os.path.abspath(path_or_file)</pre>
<pre class="line before"><span class="ws"></span> </pre>
<pre class="line current"><span class="ws">        </span>stat = os.stat(path)</pre>
<pre class="line after"><span class="ws">        </span>size = stat.st_size</pre>
<pre class="line after"><span class="ws">        </span>mtime = stat.st_mtime</pre>
<pre class="line after"><span class="ws">    </span>else:</pre>
<pre class="line after"><span class="ws">        </span>file = path_or_file</pre>
<pre class="line after"><span class="ws"></span> </pre></div>
</div>
</ul>
  <blockquote>FileNotFoundError: [Errno 2] No such file or directory: &#39;/Users/stuff/Desktop/Files/project/vton/fifa-demo/fifa-demo/results/test/try-on/tryon_with_bg.png&#39;
</blockquote>
</div>

<div class="plain">
    <p>
      This is the Copy/Paste friendly version of the traceback.
    </p>
    <textarea cols="50" rows="10" name="code" readonly>Traceback (most recent call last):
  File &#34;/Users/stuff/anaconda3/envs/image_based_vton/lib/python3.8/site-packages/flask/app.py&#34;, line 2551, in __call__
    return self.wsgi_app(environ, start_response)
  File &#34;/Users/stuff/anaconda3/envs/image_based_vton/lib/python3.8/site-packages/flask/app.py&#34;, line 2531, in wsgi_app
    response = self.handle_exception(e)
  File &#34;/Users/stuff/anaconda3/envs/image_based_vton/lib/python3.8/site-packages/flask/app.py&#34;, line 2528, in wsgi_app
    response = self.full_dispatch_request()
  File &#34;/Users/stuff/anaconda3/envs/image_based_vton/lib/python3.8/site-packages/flask/app.py&#34;, line 1825, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File &#34;/Users/stuff/anaconda3/envs/image_based_vton/lib/python3.8/site-packages/flask/app.py&#34;, line 1823, in full_dispatch_request
    rv = self.dispatch_request()
  File &#34;/Users/stuff/anaconda3/envs/image_based_vton/lib/python3.8/site-packages/flask/app.py&#34;, line 1799, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File &#34;/Users/stuff/Desktop/Files/project/vton/fifa-demo/app.py&#34;, line 367, in virtual_tryon
    response2 = send_file(
  File &#34;/Users/stuff/anaconda3/envs/image_based_vton/lib/python3.8/site-packages/flask/helpers.py&#34;, line 537, in send_file
    return werkzeug.utils.send_file(  # type: ignore[return-value]
  File &#34;/Users/stuff/anaconda3/envs/image_based_vton/lib/python3.8/site-packages/werkzeug/utils.py&#34;, line 440, in send_file
    stat = os.stat(path)
FileNotFoundError: [Errno 2] No such file or directory: &#39;/Users/stuff/Desktop/Files/project/vton/fifa-demo/fifa-demo/results/test/try-on/tryon_with_bg.png&#39;
</textarea>
</div>
<div class="explanation">
  The debugger caught an exception in your WSGI application.  You can now
  look at the traceback which led to the error.  <span class="nojavascript">
  If you enable JavaScript you can also use additional features such as code
  execution (if the evalex feature is enabled), automatic pasting of the
  exceptions and much more.</span>
</div>
      <div class="footer">
        Brought to you by <strong class="arthur">DON'T PANIC</strong>, your
        friendly Werkzeug powered traceback interpreter.
      </div>
    </div>

    <div class="pin-prompt">
      <div class="inner">
        <h3>Console Locked</h3>
        <p>
          The console is locked and needs to be unlocked by entering the PIN.
          You can find the PIN printed out on the standard output of your
          shell that runs the server.
        <form>
          <p>PIN:
            <input type=text name=pin size=14>
            <input type=submit name=btn value="Confirm Pin">
        </form>
      </div>
    </div>
  </body>
</html>

<!--

Traceback (most recent call last):
  File "/Users/stuff/anaconda3/envs/image_based_vton/lib/python3.8/site-packages/flask/app.py", line 2551, in __call__
    return self.wsgi_app(environ, start_response)
  File "/Users/stuff/anaconda3/envs/image_based_vton/lib/python3.8/site-packages/flask/app.py", line 2531, in wsgi_app
    response = self.handle_exception(e)
  File "/Users/stuff/anaconda3/envs/image_based_vton/lib/python3.8/site-packages/flask/app.py", line 2528, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/stuff/anaconda3/envs/image_based_vton/lib/python3.8/site-packages/flask/app.py", line 1825, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/stuff/anaconda3/envs/image_based_vton/lib/python3.8/site-packages/flask/app.py", line 1823, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/stuff/anaconda3/envs/image_based_vton/lib/python3.8/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/Users/stuff/Desktop/Files/project/vton/fifa-demo/app.py", line 367, in virtual_tryon
    response2 = send_file(
  File "/Users/stuff/anaconda3/envs/image_based_vton/lib/python3.8/site-packages/flask/helpers.py", line 537, in send_file
    return werkzeug.utils.send_file(  # type: ignore[return-value]
  File "/Users/stuff/anaconda3/envs/image_based_vton/lib/python3.8/site-packages/werkzeug/utils.py", line 440, in send_file
    stat = os.stat(path)
FileNotFoundError: [Errno 2] No such file or directory: '/Users/stuff/Desktop/Files/project/vton/fifa-demo/fifa-demo/results/test/try-on/tryon_with_bg.png'


-->
